src/mono/wasm/debugger/tests/debugger-test/debugger-test.csproj

```diff
@@ -22,6 +22,7 @@
     <ProjectReference Include="..\library-dependency-debugger-test2\library-dependency-debugger-test2.csproj" Private="true"/>
     <ProjectReference Include="..\debugger-test-with-source-link\debugger-test-with-source-link.csproj" Private="true"/>
     <ProjectReference Include="..\ApplyUpdateReferencedAssembly\ApplyUpdateReferencedAssembly.csproj" />
+    <ProjectReference Include="..\debugger-test-with-full-debug-type\debugger-test-with-full-debug-type.csproj" Private="true"/>
```

Comment:
does this need to be added?

Reply:
I think so, to compile the new project for the test case.

Reply:
right, I mean did you forget to add the project to the pr?

Reply:
I'm sorry Larry!